app/app-controllers/searchBarController.js

```diff
@@ -1,14 +1,17 @@
 define([
     'bluebird',
-    'plugins/webViewPlugin'
+    'plugins/searchBarPlugin',
+    'plugins/anchoredLayoutPlugin'
 ],
 function(
     Promise,
-    WebViewPlugin
+    SearchBarPlugin,
+    AnchoredLayoutPlugin
 ) {
-    var SearchBarController = function(webView, layout, searchConfig) {
-        this.viewPlugin = webView;
+    var SearchBarController = function(searchBarPlugin, layout, internalLayout, searchConfig) {
+        this.viewPlugin = internalLayout;
         this.layout = layout;
```

Comment:
Let's rename this to make it more obvious that it is the external/parent layout that we need to add ourselves to. Maybe `parentLayout` ?